probe/probe.go

```diff
@@ -590,18 +599,18 @@ func getAnnotatedPods(c *k8s.Client) error {
 			if err != nil {
 				return err
 			}
+			var labels []string
 			data = append(data, []string{armoredPod.Namespace, armoredPod.Name, ""})
-			for key, value := range armoredPod.Labels {
-				for policyKey, policyValue := range policyMap {
-					if slices.Contains(policyValue, key+":"+value) {
-						if checkIfDataAlreadyContainsPodName(data, armoredPod.Name, policyKey) {
-							continue
-						} else {
-							data = append(data, []string{armoredPod.Namespace, armoredPod.Name, policyKey})
-						}
-
+			labels = getAnnotatedPodLabels(armoredPod.Labels, labels)
```

Comment:
Can we directly return a mapset from the getAnnotatedPodLabels()? Rather than returning first a string slice (labels( and then converting the slice to mapset (s1).